symplyphysics/definitions/volume_charge_density_from_charge_and_volume.py

```diff
@@ -0,0 +1,37 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+## The volume charge density is defined as the amount of charge present over a unit volume of the conductor.
+## Charge density can be either positive or negative, since electric charge can be either positive or negative.
+
+
+## Definition: ρ = q / V
```

Comment:
Derive this law from https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/volume_number_density.py

Reply:
This is a definition, not a law, it is not derived from anywhere

Reply:
Yes, but obviously they are the same formula. So we can say, that by definition, volumetric charge density is volumetric density (which we have a definition of N/V) of charges. And this specific formula in your law is derived from the definitions of volumetric density and of volumetric charge density.